com.unity.perception/Editor/Randomization/Editors/PerceptionEditorAnalytics.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using JetBrains.Annotations;
+using UnityEditor;
+using UnityEngine.Analytics;
+
+namespace UnityEngine.Perception.Randomization.Editor
+{
+    static class PerceptionEditorAnalytics
+    {
+        static int k_MaxItems = 100;
+        static int k_MaxEventsPerHour = 100;
+        const string k_VendorKey = "unity.perception";
+        const string k_RunInUnitySimulationName = "runinunitysimulation";
+
+        static bool k_IsRegistered = false;
```

Comment:
My Rider seems to think this line should be:
`static bool s_IsRegistered;`

Reply:
Good catch. Addressed.